src/DataValues/MonolingualTextValue.php

```diff
@@ -277,7 +289,12 @@ public function getTextValueByLanguageCode( $languageCode ) {
 			return null;
 		}
 
-		if ( $list['_LCODE'] !== Localizer::asBCP47FormattedLanguageCode( $languageCode ) ) {
+		$mappedLanguageCode = !array_key_exists( $list['_LCODE'], $this->nonstandardLanguageCodeMapping ) ? $list['_LCODE']
```

Comment:
More idiomatic way to write this is `$mappedLanguageCode = $this->nonstandardLanguageCodeMapping[$list['_LCODE']] ?? $list['_LCODE']`